File: library-backend/routes/authRoutes.js
// routes/authRoutes.js
const express = require('express');
const router = express.Router();
const authController = require('../controllers/authController');

// 注册 routes/authRoutes.js
router.post('/register', authController.register);
// 登录 routes/authRoutes.js
router.post('/login', authController.login);

module.exports = router;

File: library-backend/routes/authRoutes.go
package routes

import (
	"library-backend/controllers"
	"net/http"

	"github.com/gorilla/mux"
)

// AuthRoutes 设置认证相关的路由 / Set authentication-related routes
func AuthRoutes(router *mux.Router) {

	router.HandleFunc("/register", controllers.RegisterUser).Methods(http.MethodPost)
	router.HandleFunc("/login", controllers.LoginUser).Methods(http.MethodPost)
}
